test-sbt-testing/js/src/test/scala/zio/test/runner/DummyJSSpec.scala

```diff
@@ -0,0 +1,14 @@
+package zio.test.runner
```

Comment:
I'd add this to the top-level `examples` project that depends on both ZIO Core and ZIO Tests. It can have examples of both ZIO Test and ZIO proper.

Reply:
The idea for this project was to be able to manually run `testRunnerTestingJVM/test` to test the sbt runner.
I guess I could to the same on the examples project...